modules/libgcrypt

```diff
@@ -14,7 +14,6 @@ libgcrypt_configure := \
 	--prefix "/" \
 	--disable-static \
 	--with-gpg-error-prefix="$(INSTALL)" \
-	--disable-asm \
```

Comment:
Summarizing a discussion via DM - I'm good with this change.

And odd thing sticking out to me was that libgcrypt has code to use RDRAND directly for up to 25% of its entropy, which would have been prevented in the past with `--disable-asm`.  That doesn't make much sense with the kernel already including RDSEED in its CRNG, and it doesn't even know about things like the broken Ryzen microcode, etc.  But this code is no longer active by default, that code is for USE_RNDOLDLINUX, default is now USE_RNDGETENTROPY.

So no issues here, let's keep this change.